.github/scripts/build-tflocal/main.go

```diff
@@ -45,9 +44,7 @@ func main() {
 		log.Fatal(err)
 	}
 
-	if err := waitForRun(ctx, client, runID); err != nil {
-		log.Fatal(err)
-	}
+	log.Printf(runID)
```

Comment:
is there a reason for printing just the runID here?

Reply:
We don't necessarily have to, but it makes it easier to troubleshoot and associate a workflow to a specific run.